tests/integration_test.rs

```diff
@@ -31,7 +31,7 @@ fn test_end2end_execution() {
     assert!(tmp_dir_path.exists());
 
     let files = std::fs::read_dir(tmp_dir_path).unwrap();
-    assert_eq!(846, files.into_iter().collect::<Vec<_>>().len());
+    assert_eq!(686, files.into_iter().collect::<Vec<_>>().len());
```

Comment:
This is because the mock integration test infra was automatically considering every multiversion suite as a ReplicaSet, which would mean every multiversion task would generate many tasks. Now that the generated tasks are explicitly written in the evergreen.yml, that "mock" codepath is not used & the number of files is significantly less.